README.md

```diff
@@ -137,6 +137,17 @@ From your terminal, navigate to the directory you'd like to start from and run:
 goose session start
 ```
 
+#### Set up a provider
+Goose works on top of LLMs and by default, it uses `openai` as the LLM provider but you can customize it as needed. You'll be prompted to set an [OPENAI_API_KEY][openai-key] if you haven't set one previously.
```

Comment:
```suggestion
Goose works with your [preferred LLM](https://block.github.io/goose/plugins/providers.html). By default, it uses `openai` as the LLM provider. You'll be prompted to set an [API key][openai-key] if you haven't set one previously.
```

rephrasing a bit to let them know other LLMs are available. Don't want people to be DOA if they use something other than Open AI